contracts/MemeFactory.sol

```diff
@@ -155,27 +160,32 @@ abstract contract MemeFactory {
 
     /// @dev Creates reference token + meme token LP and adds liquidity.
     /// @param memeToken Meme token address.
-    /// @param referenceTokenAmount Reference token amount.
+    /// @param nativeTokenAmount Native token amount.
     /// @param memeTokenAmount Meme token amount.
     /// @return pair reference token + meme token LP address.
     /// @return liquidity Obtained LP liquidity.
     function _createUniswapPair(
         address memeToken,
-        uint256 referenceTokenAmount,
+        uint256 nativeTokenAmount,
         uint256 memeTokenAmount
     ) internal returns (address pair, uint256 liquidity) {
-        // Create the LP
-        pair = IUniswap(factory).createPair(referenceToken, memeToken);
+        _wrap(nativeTokenAmount);
```

Comment:
I don't think you're using _wrap consistently. You don't have this as a private method in BaseMeme

Reply:
I'm inheriting it without any change, so it's good to go.